src/Microsoft.Data.SqlClient/tests/tools/Microsoft.Data.SqlClient.TestUtilities/Fixtures/CertificateFixtureBase.cs

```diff
@@ -54,7 +56,7 @@ protected static X509Certificate2 CreateCertificate(string subjectName, IEnumera
 #if NET9_0_OR_GREATER
```

Comment:
```suggestion
#if NET
```
The .NET 8 target on Ubuntu is trying to use the Windows-specific path. This should resolve that.